docs/framework/configure-apps/file-schema/appsettings/add-element-for-appsettings.md

```diff
@@ -54,6 +54,15 @@ The following example shows how to add a custom configuration setting for the ap
 </appSettings>
 ```
 
+The following example uses the `<add>` element to define two compatibility settings in an ASP.NET application:
+
+```xml
+<appSettings>
+  <add key="Switch.System.,Globalization.NoAsyncCurrentCulture" value="true" />
```

Comment:
This needs to be: 
key="AppContext.SetSwitch:Switch.System.Globalization.NoAsyncCurrentCulture" value="true" />

Same for the one below.